src/vivarium_csu_alzheimers/components/population.py

```diff
@@ -131,7 +131,7 @@ def on_time_step(self, event: Event) -> None:
     ##################
 
     def load_incidence_rate(self, builder: Builder) -> pd.Series:
-        incidence_rate = builder.data.load(data_keys.ALZHEIMERS.INCIDENCE_RATE)
+        incidence_rate = builder.data.load(data_keys.ALZHEIMERS.RAW_INCIDENCE)
```

Comment:
for me, it would be clearer to call this "TOTAL_POPULATION_INCIDENCE_RATE", because it is hard to know what raw is and what it is supposed to be.